packages/template-retail-react-app/app/components/list-menu/index.jsx

```diff
@@ -80,7 +79,7 @@ const ListMenuTrigger = ({item, name, isOpen, onOpen, onClose, hasItems}) => {
                 {...baseStyle.listMenuTriggerLinkIcon}
             >
                 <PopoverTrigger>
-                    <Fade in={hasItems && item.loaded}>
```

Comment:
Via the level zero categories call, you won't have enough info to know whether you need to fade this in or not. So `hasItems` is insufficient here